test/qix-create-reload-doc.spec.js

```diff
@@ -53,26 +59,40 @@ describe('QIX doc sync - reload doc', () => {
   it('should be possible to create a doc in one engine and reload in another', async () => {
     // Open a session to first engine and create an app
     console.log('Opening a session to first qix-engine');
-    const session = enigma.create({
+    const session1 = enigma.create({
       schema,
       url: `ws://${engine1}/app/engineData/identity/${+new Date()}`,
       createSocket: url => new WebSocket(url),
     });
 
-    const qix1 = await session.open();
-    console.log('Creating doc');
-    const result = await qix1.createApp('testApp');
+    session1.on("traffic:*", (direction, data) => console.log("session1", direction, JSON.stringify(data)));
+
+    const qix1 = await session1.open();
+    console.log('Creating doc in first qix-engine instance');
+    const result = await qix1.createApp(`testApp${+new Date()}`);
     expect(result.qSuccess).to.equal(true);
     console.log(`Created doc with id: ${result.qAppId}`);
     const appId = result.qAppId;
 
     // Open doc and save the reload time
-    let app = await qix1.openDoc(appId);
-    let appLayout = await app.getAppLayout();
-    const firstReloadTime = appLayout.qLastReloadTime;
-    console.log(`Doc was reloaded last time: ${firstReloadTime}`);
-    await session.close();
-    console.log('Closed session to first qix-engine');
+    const app = await qix1.openDoc(appId);
+    const appLayout = await app.getAppLayout();
+    await app.doSave();
+    await sleep(1000);
```

Comment:
Why are we sleeping here?

Reply:
To get a document timestamp that differs more than a second i.e. next `DoSave` on the document is not within the same second.